src/um/wtsapi32.rs

```diff
@@ -0,0 +1,925 @@
+ // Copyright © 2015-2018 winapi-rs developers
+// Licensed under the Apache License, Version 2.0
+// <LICENSE-APACHE or http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your option.
+// All files in the project carrying such notice may not be copied, modified, or distributed
+// except according to those terms.
+use shared::minwindef::{BOOL, BYTE, DWORD, LPDWORD, MAX_PATH, PBOOL, PULONG, ULONG, USHORT};
+use shared::windef::HWND;
+use um::winnt::{
+    CHAR, HANDLE, LARGE_INTEGER, LONG, LPSTR, LPWSTR, PCHAR, PSECURITY_DESCRIPTOR, PSID, PSTR,
+    PVOID, PWSTR, SECURITY_INFORMATION, STANDARD_RIGHTS_REQUIRED, WCHAR
+};
+pub const WTS_CURRENT_SERVER: HANDLE = 0isize as HANDLE;
+pub const WTS_CURRENT_SERVER_HANDLE: HANDLE = 0isize as HANDLE;
```

Comment:
Shouldn't `WTS_CURRENT_SERVER_NAME` be right here? ^^

Reply:
This one I didn't know what type to give it, it's defined as NULL, it's going to be either an LPSTR or a LPWSTR. Maybe for this one we could add WTS_CURRENT_SERVER_NAME_A and WTS_CURRENT_SERVER_NAME_W?

Reply:
Just put it in as a comment for now so I can decide what to do with it later.

Reply:
Done.